doc/variableexplorer.rst

```diff
@@ -2,95 +2,213 @@
+
+.. image:: images/variable_explorer/variable-explorer-modifying.gif
+   :align: center
+   :alt: Spyder Variable Explorer modifying value of a variable 
+
+|
+
+==============
+Object Viewers
+==============
+
+Spyder's :guilabel:`Variable Explorer` offers built in support for editing lists, strings, dictionaries, NumPy arrays, Pandas DataFrames, and more, and can also histogram, plot, or even display some of them as an RGB image. Several examples of this functionality are shown below:
 
-Spyder's :guilabel:`Variable Explorer` offers built in support for editing lists, strings, dictionaries, NumPy arrays, Pandas DataFrames, and more, and can also histogram, plot, or even display some of them as an RGB image.
-Several examples of this functionality follow:
 
 |
 
-.. image:: images/variable_explorer/variable_explorer_text_long.png
+~~~~~~~
+Strings
+~~~~~~~
+
+When a string variable is longer than forty characters, you can double click it to see its value in a text editor where you can modify it.
+
+.. image:: images/variable_explorer/variable-explorer-text-long.png
    :align: center
    :alt: Variable Explorer text editor, displaying a long string in a window
 
 |
 
-.. image:: images/dialog/dialog_user_env_variables_edit.png
+~~~~~~~~~~~~
+Dictionaries
+~~~~~~~~~~~~
+
+When the type of a variable is a dictionary, double clicking it will show you a viewer displaying each one of the keys of the dictionary along with its associated value. You can double click any of the values to modify it. 
+
+.. image:: images/variable_explorer/variable-explorer-dictionary.png
    :align: center
    :alt: Dictionary editor displaying keys and their types, sizes, and values
 
 |
 
-.. image:: images/variable_explorer/variable_explorer_list_timedelta_edit.png
+~~~~~
+Lists
+~~~~~
+
+For variables that are of type list, the window pane only displays the first ten values of the list. If you want to see them all, double click the variable in the pane to open a viewer that will display the index, type, size and value of each element of the list. You can also modify any value by double clicking it in the viewer and writing the new one.
+
+.. image:: images/variable_explorer/variable-explorer-list.png
    :align: center
-   :alt: List editor displaying a list of timedeltas, showing one being edited
+   :alt: List editor displaying a list, showing one being edited
 
 |
 
-.. image:: images/variable_explorer/variable_explorer_array_2D_resize.png
+~~~~~~~~~~~~
+Numpy Arrays
+~~~~~~~~~~~~
+
+For Numpy Arrays, the window pane only shows some of the values of the array. You can open a viewer that shows you graphically in a "heat map" the values of the array, displaying each value in a colored box, depending on the size of the value. You can deactivate the background color by unchecking the option in the window. If the array is too large, background color will be automatically deactivated.
```

Comment:
```suggestion
For Numpy Arrays, the Variable Explorer only shows some of their values. By double-clicking on them, a viewer will be shown that displays graphically the array values in a "heat map", i.e. it displays each value in a colored box, depending on the size of the value. You can deactivate the background color by unchecking the option in the viewer. If the array is too large, the background color will be automatically deactivated.
```

Reply:
I think it will be better to say "By double-clicking them, a viewer will be shown displaying graphically the array values in a "heat map" but @CAM-Gerlach, what do you think?

Reply:
I think it will be better to say "By double-clicking them, a viewer will be shown graphically displaying the array values in a "heat map" but @CAM-Gerlach, what do you think?

Reply:
By double-clicking them, a viewer will be shown graphically displaying the array values in a "heat map

Reply:
Ok, I changed the text to what you suggested.